src/test/java/com/thebois/models/beings/BeingTests.java

```diff
@@ -160,4 +183,79 @@ public void equalsReturnsFalseForNotEqualBeings(final AbstractBeing first,
         assertThat(first).isNotEqualTo(second);
     }
 
+    @Test
+    public void pawnBuildsHouses() {
```

Comment:
Generally it's helpful if the test names follow the format:
`whenSomethingIsDoneAndTheObjectIsInThisStateThenThisHappens`
On a lot of these tests, you've only got the:
`ThisHappens`, you never describe what is being done and what state the object is in.

Now, this is easy to see if I read the code of the test, but from the test runner all you can see is the name and whether it failed or not, so it's much more helpful to read that "`updateDeliversItemsToStructureWhenNearby` has failed" rather than
"`pawnBuildsHouses` has failed", if that makes sense?